src/components/Collections/CollectionInfo.tsx

```diff
@@ -11,13 +11,25 @@ interface Iprops {
 export function CollectionInfo({ name, fileCount, endIcon }: Iprops) {
     return (
         <div>
+            <Typography variant="h3">{name}</Typography>
```

Comment:
I have forgotten, why there is that extra div, can we just put everything inside the FlexWrapper and remove the extra div

Reply:
FlexWrapper will wrap 'Album name' and 'album memory count' in the same line which we don't want
![Screenshot from 2023-01-20 16-58-35](https://user-images.githubusercontent.com/55018937/213684947-79f2f571-36ab-4e16-8666-726aa2636667.png)

Reply:
ohh okk okk